utils/reviewsUtils.js

```diff
@@ -50,5 +50,13 @@ export function filterVerified(arr){
 
 //skriv här
 
-
-
+export async function postReview(review) {
+    const res = await fetch(`${API_BASE}/reviews`, {
+      method: 'POST',
```

Comment:
Indentation here is a bit sloppy. The first line has no indentation, the second has 4 spaces, and the third has 6 spaces. This means that the levels are not consistent. I would expect 0+2+4 or 0+4+8.